### ISSUE TITLE
style dont seems to work ?

### ISSUE THREAD

**Issue by rhgiter** [nervetattoo/simple-thermostat#280]
![image](https://user-images.githubusercontent.com/61065752/137619054-e3572a86-7c5f-46e1-92b9-68e999480b20.png)

**Comment by shadowdavis** [comment]
me too..why?

**Comment by Normanras** [comment]
Did you install [card-mod](https://github.com/thomasloven/lovelace-card-mod) yet via HACS? The documentation says you can edit simple-thermostat css with card-mod, so double-check that you've already installed that.

**Comment by rhgiter** [comment]
ahh - I missed that. Now it works on pc:-) , but It dont show on my phone. ? do you know why?

**Comment by Normanras** [comment]
Sorry for the delay! Could be a number of reasons. card-mod doesn't show up on my old amazon fire tablets. What phone are you using?

**Comment by rhgiter** [comment]
samsung s10e

**Comment by Normanras** [comment]
Yeah that's strange since its a fairly new phone. I would make sure your OS and HA app are all up to date. If it still isn't working, and you've confirmed it works on your computer's browser or on another device, [try this thread.](https://www.reddit.com/r/homeassistant/comments/ly3xw9/cardmod_v30_is_not_working_for_me/) It suggests making sure you have up to date card-mod in your dev tools.
